src/Preamble.fun

```diff
@@ -1,5 +1,6 @@
 functor Preamble (
   val preamble : string
+  val show : Rational.t -> bool
```

Comment:
One thought: instead of `Rational.t`, what about `Grader.Rubric.t`, so the preamble switch doesn't have to factor through the score?

Upon further reflection, perhaps we should even generalize to `val show : Grader.Rubric.t -> string option` or something, allowing for a different preamble per rubric item? (The existing behavior could be recovered with `Fn.const (SOME preambleString)`, of course.)

Reply:
> I don't think we should merge this as-is

Fair

> `val show : Grader.Rubric.t -> string option` or something, allowing for a different preamble per rubric item?

I thought about this but the issue I see with it is that we need to know the what the rubric is which isn't exactly clear and would make it hard to actually use this.